package.json

```diff
@@ -1,18 +1,22 @@
 {
   "name": "react-overdrive",
-  "version": "0.0.10",
+  "version": "0.0.11",
   "description": "Super easy magic-move transitions for React apps",
   "main": "lib/Overdrive.min.js",
   "files": [
     "lib",
-    "src"
+    "src",
+    "index.d.ts"
   ],
   "scripts": {
     "build": "standard && npm test && NODE_ENV=production webpack --env build",
     "dev": "webpack --progress --colors --watch --env dev",
     "test": "jest",
     "standard": "standard"
   },
+  "dependencies": {
+    "@types/react": "^15.5.0 || ^16.0.0"
```

Comment:
For people who don't use TypeScript, what's the best practice of excluding this from the bundle?

Reply:
🤔 that's a good one, I believe that it'd be better that we move this into peer dependencies

Reply:
Well, actually... I believe that I can add a section in the `readme.md` file for people using TypeScript to be aware that a type definition is needed, so that non-TS users don't see any warnings from NPM for a missing peer dependency. Thoughts?